build_unix/build.sh

```diff
@@ -4,3 +4,8 @@ cd build
 cmake ..
 make
 cd ..
+mkdir -p build-ui/
+~/Qt5.6.*/5.6*/gcc*/bin/qmake -o build-ui/
```

Comment:
Why isn't this only `qmake`? If you install Qt, `qmake` should be available as a global command.